problems/secondSmallest.js

```diff
@@ -6,8 +6,10 @@
 * @returns {number} - Returns the second smallest number.
 */
 
-function secondSmallest() {
-  
-}
+function secondSmallest(arr) {
+arr.sort((a, b) => {
+return a - b;
+});
```

Comment:
This sorts the array, but we still need to return the second smallest. return arr[1]. 
We can also do this by keeping track of two variables, smallest and secondSmallest and updating them as we iterate through the array.